website/docs/docs/cloud/about-cloud-develop.md

```diff
@@ -0,0 +1,19 @@
+---
+title: About developing in dbt Cloud
+id: about-cloud-develop
+description: "Learn how to develop your dbt projects using dbt Cloud."
+sidebar_label: "About developing in dbt Cloud"
+pagination_next: "docs/cloud/cloud-cli-installation"
+---
+
+dbt Cloud offers a fast and reliable way to work on your dbt project. You can use it through an integrated development environment (IDE) in your browser or a dbt Cloud-powered command line interface (CLI):
+
+- **[dbt Cloud CLI](/docs/cloud/cloud-cli-installation)** &mdash; Allows you to develop and run dbt commands from your local command line or code editor against your dbt Cloud development environment.
+  - Note, the dbt Cloud CLI is distinct from [dbt Core](https://github.com/dbt-labs/dbt-core), an open-source command-line tool for dbt. Both tools are command lines that use the same dbt commands, however, the dbt Cloud CLI is tailored to run specifically on dbt Cloud's infrastructure.
```

Comment:
is this clear @greg-mckeon @jtcohen6 ?

```suggestion
  - The dbt Cloud CLI and [dbt Core](https://github.com/dbt-labs/dbt-core), an open-source project, are both command line tools that use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure.
```

Reply:
I would either leave this off, or clarify the value:

`The dbt Cloud CLI and [dbt Core](https://github.com/dbt-labs/dbt-core), an open-source project, are both command line tools that run the same dbt commands. The dbt Cloud CLI  integrates with your dbt Cloud development environment, providing faster and cheaper builds, centralized access management and access to all dbt Cloud features.`

Reply:
ok gotcha! i'm just trying to make sure users don't get confused with the 'cli' term we use and associate with dbt core, and the dbt cloud cli.